datasafe-business/src/test/java/de/adorsys/datasafe/business/impl/e2e/BaseStorageTest.java

```diff
@@ -89,4 +102,116 @@ private void validatePrivateStructAndEncryption(AbsoluteResourceLocation<Private
                 .filter(it -> pattern.test(it.location().toString()))
                 .collect(Collectors.toList());
     }
+
+    @Test
+    @SneakyThrows
+    @Ignore
+    public void WriteToPrivateListPrivateInDifferentThreads() {
+        CountDownLatch fileSaveCountDown = new CountDownLatch(NUMBER_OF_TEST_USERS * NUMBER_OF_TEST_FILES);
+        String path = "folder2";
+
+        log.trace("*** Starting write threads ***");
+        for (int i = 0; i < NUMBER_OF_TEST_USERS; i++) {
+            UserIDAuth john = registerUser("john_" + i, location);
+            log.debug("Registered user: {}", john.getUserID().getValue());
+
+            AtomicInteger counter = new AtomicInteger();
+
+            for (int j = 0; j < NUMBER_OF_TEST_FILES; j++) {
+                new Thread(() -> {
+                    try {
+                        log.trace("Start thread: {} for user: {}",
```

Comment:
Instead it makes sense to set thread name equal to user name via 
```Thread.currentThread().setName(userName)```
so that we can follow whole user flow in logs

Reply:
ok